tests/components/zwave_js/test_device_condition.py

```diff
@@ -430,7 +430,18 @@ async def test_get_condition_capabilities_value(
     )
     assert capabilities and "extra_fields" in capabilities
 
-    cc_options = [(cc.value, cc.name) for cc in CommandClass]
+    cc_options = [
+        (133, "ASSOCIATION"),
+        (128, "BATTERY"),
+        (112, "CONFIGURATION"),
+        (98, "DOOR_LOCK"),
+        (122, "FIRMWARE_UPDATE_MD"),
+        (114, "MANUFACTURER_SPECIFIC"),
+        (113, "ALARM"),
```

Comment:
Should this be updated to `(113, "NOTIFICATION")` when we bump the library?

Reply:
yes, I will have to update this as well as some other tests when we do the bump